lib/appConfig.ts

```diff
@@ -221,7 +221,7 @@ export const createAppConfig = (entries: { [entryAlias: string]: string }, optio
 								return `js/${assetsPrefix}[name].mjs`
 							},
 							chunkFileNames: () => {
-								return 'js/[name].chunk.mjs'
+								return 'js/[name]-[hash].mjs'
```

Comment:
We could also cleanup the directory (if someone cares) with:
```suggestion
								return 'js/chunks/[name]-[hash].mjs'
```

Reply:
Makes sense, let me push that here as well

Reply:
Or `js/[name]-[hash].chunk.mjs` to be consistant with css chunks?

Reply:
btw, won't we have the same issue with CSS chunk?
https://github.com/nextcloud-libraries/nextcloud-vite-config/blob/main/lib/plugins/CSSEntryPoints.ts#L49